src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -80,13 +81,14 @@ const searchPageTitle = (
 const defaultSearchParams: PeopleSearchQuery = {
   includeStudent: true,
   includeFacStaff: true,
-  includeAlumni: false,
+  includeAlumni: true,
```

Comment:
I think this should stay as false. I get that it's confusing that you need to check this and also enter a year, but we want to show only current community members unless the user requests alumni.